ui_src/src/domain/schema/components/schemaDetails/index.js

```diff
@@ -12,22 +12,22 @@
 
 import './style.scss';
 
-import { AddRounded, CheckCircleOutlineRounded, ErrorOutlineRounded } from '@material-ui/icons';
+import { AddRounded, CheckCircleOutlineRounded, Create, ErrorOutlineRounded } from '@material-ui/icons';
```

Comment:
Unneccerry import

Reply:
btw I have not added this part but I will remove it